2021-01-23 13:59:52,115 INFO    MainThread:7799 [wandb_setup.py:_flush():70] setting env: {}
2021-01-23 13:59:52,115 INFO    MainThread:7799 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-23 13:59:52,115 INFO    MainThread:7799 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-23 13:59:52,115 INFO    MainThread:7799 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-23 13:59:52,115 INFO    MainThread:7799 [wandb_init.py:_log_setup():318] Logging user logs to /home/iiitd/Desktop/Ahmad/doa1/wandb/run-20210123_135952-50wmygvx/logs/debug.log
2021-01-23 13:59:52,115 INFO    MainThread:7799 [wandb_init.py:_log_setup():319] Logging internal logs to /home/iiitd/Desktop/Ahmad/doa1/wandb/run-20210123_135952-50wmygvx/logs/debug-internal.log
2021-01-23 13:59:55,363 INFO    MainThread:7799 [wandb_run.py:_console_start():1308] atexit reg
2021-01-23 13:59:55,364 INFO    MainThread:7799 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-23 13:59:55,364 INFO    MainThread:7799 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-23 13:59:55,364 INFO    MainThread:7799 [redirect.py:install():205] install start
2021-01-23 13:59:55,364 INFO    MainThread:7799 [redirect.py:install():220] install stop
2021-01-23 13:59:55,364 INFO    MainThread:7799 [redirect.py:install():205] install start
2021-01-23 13:59:55,368 INFO    MainThread:7799 [redirect.py:install():220] install stop
2021-01-23 13:59:55,368 INFO    MainThread:7799 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-23 14:02:57,762 INFO    MainThread:7799 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-23 14:02:57,762 INFO    MainThread:7799 [wandb_run.py:_restore():1250] restore
2021-01-23 14:02:57,762 INFO    MainThread:7799 [redirect.py:uninstall():224] uninstall start
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:_stop():279] _stop: stdout
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:_stop():285] _stop closed: stdout
2021-01-23 14:02:57,763 INFO    stdout    :7799 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-23 14:02:57,763 INFO    stdout    :7799 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:_stop():291] _stop joined: stdout
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:uninstall():228] uninstall done
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:uninstall():224] uninstall start
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:_stop():279] _stop: stderr
2021-01-23 14:02:57,763 INFO    MainThread:7799 [redirect.py:_stop():285] _stop closed: stderr
2021-01-23 14:02:57,763 INFO    stderr    :7799 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-23 14:02:57,764 INFO    stderr    :7799 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-23 14:02:57,764 INFO    MainThread:7799 [redirect.py:_stop():291] _stop joined: stderr
2021-01-23 14:02:57,764 INFO    MainThread:7799 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-23 14:02:57,764 INFO    MainThread:7799 [redirect.py:uninstall():228] uninstall done
2021-01-23 14:02:58,081 INFO    MainThread:7799 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "26d469729a75464bb600b9d5a771bb27"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 511
      total_bytes: 511
    }
  }
}

2021-01-23 14:03:00,094 INFO    MainThread:7799 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "0e96152c63b3485ea446a1f3b11322e9"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6474
      total_bytes: 6474
    }
  }
}

2021-01-23 14:03:01,321 INFO    MainThread:7799 [wandb_run.py:_show_files():1614] logging synced files
2021-01-23 14:03:01,323 INFO    MainThread:7799 [internal.py:handle_exit():139] Internal process exited
